pipelines/managed/rh-advisories/rh-advisories.yaml

```diff
@@ -767,13 +723,10 @@ spec:
           - name: url
             value: https://github.com/konflux-ci/mobster.git
           - name: revision
-            value: $(params.mobster_tasks_git_revision)
+            value: remove-workspaces-from-tasks  #  $(params.mobster_tasks_git_revision)
```

Comment:
Why is this not in the main branch? How are we going to maintain this topic branch going forward? Or is this a draft or something temporary? It doesn't say so anywhere.

Reply:
I was going to create a PR for them once I got tests working and this idea was approved by the team.

I mentioned this in the commit msg and PR description but you are right...there is no comment in the code. I could add that.

we'll keep this open to remind me to make sure this is in main.